Source/WebCore/rendering/svg/RenderSVGResourceFilterPrimitive.cpp

```diff
@@ -61,4 +67,28 @@ void RenderSVGResourceFilterPrimitive::markFilterEffectForRebuild()
     parent->invalidateFilter();
 }
 
+SVGFilterPrimitiveStandardAttributes& RenderSVGResourceFilterPrimitive::filterPrimitiveElement() const
+{
+    return static_cast<SVGFilterPrimitiveStandardAttributes&>(RenderSVGHiddenContainer::element());
+}
+
+void RenderSVGResourceFilterPrimitive::styleDidChange(StyleDifference diff, const RenderStyle* oldStyle)
+{
+    RenderSVGHiddenContainer::styleDidChange(diff, oldStyle);
+
+    if (diff == StyleDifference::Equal || !oldStyle)
+        return;
+
+    Ref newStyle = style().svgStyle();
+    if (is<SVGFEFloodElement>(filterPrimitiveElement()) || is<SVGFEDropShadowElement>(filterPrimitiveElement())) {
```

Comment:
I'd turn these into early returns, instead of `else if` constructs for readability.

Reply:
Agreed, reads a bit better, fixed.